packages/infrastructure-lib/src/mongo_settlement_transfer_repo.ts

```diff
@@ -109,6 +112,62 @@ export class MongoSettlementTransferRepo implements ISettlementBatchTransferRepo
 		}
 	}
 
+	async getBatchTransfersByBatchIdsWithPagi(
```

Comment:
Same as above, use transfers implementation as reference (2 queries, not an aggregate query)